src/Reducers/initialState.js

```diff
@@ -33,7 +33,7 @@ const initialState = {
       'Domain',
       'Owner',
       'Contains',
-      'Doesn\'t contain',
+      "Doesn't contain",
```

Comment:
this seems to be a linter issue across all of our environments. I believe that backslash needs to be there. I'm wondering if theres a prettier config option (or even, just saying "Does not contain" instead) so that it doesn't do this all the time.

Reply:
Agreed! I think prettier can be manually configured in VSCode.

That said, this prettier version should technically still work, albeit inconsistently with the rest of the string. Pretty sure you can avoid escaping apostrophes with back-ticks as well. Maybe we can set a best practice and choose one preferred method?

Reply:
For sure! I think the double quotes w/ out the escaped character would be ok, but if we really wanted to keep the single quotes, just adding `// prettier-ignore` to the line above will keep it escaped. I have it like that in my MR #39

Reply:
Yeah its linter issue. Thanks @abbyjb  
Btw I need to move those into container. its unnecessary to stay here.

Reply:
do you mean move the definition them to `mapStateToProps`? If thats the case, I don't think that should be in that method. `mapStateToProps` should only be concerned with the redux store. Since these values are never being update, we can define those within the component itself.

Reply:
Im sorry I meant to say component not container.

Reply:
<img width="731" alt="Screen Shot 2019-12-11 at 11 07 44 AM" src="https://user-images.githubusercontent.com/57808568/70652678-ddd08300-1c07-11ea-988f-34579602390e.png">
Here is the error for saga @abbyjb

Reply:
@SelcukEdma thats what I thought.... I believe we should be creating the notification message via the front end, and not though the actual response that the backend gives us. I think a good example to start from would be the `setFooterButtonText` in the `RequestInbox.js`

Reply:
@abbyjb 
Well we need to success message from the API. not the failure.Saga handles the failure. User needs to know how many assets they requested it. Thats what api returning to us. 
im sorry what is the `setFooterButtonText` for?

Reply:
I guess getting the message from the API is fine for MVP. Just wanted something more human readable for the error, like "Failed to make request".

`setFooterButtonText` is for the button at the bottom of the `RequestsTable`. Since the action of the button changes per table, `setFooterButtonText` is used to change the text on the button depending.

Reply:
I see I will ask to @sfowles about it. How he want to error to appear. I can add the text np. 
`setFooterButtonText` is fucntion for generate dynamic text for ur table. But button has a static text doesnt need any changes. If you are talking generic button component, I am planing to make generic button component but idk is it worth to implement